src/AccessibilityInsights.SharedUx/Properties/AutomationIDs.cs

```diff
@@ -4,11 +4,19 @@ namespace AccessibilityInsights.SharedUx.Properties
 {
     public static class AutomationIDs
     {
+        public static string AutomatedChecksExpandAllButton{ get; } = nameof(AutomatedChecksExpandAllButton);
```

Comment:
>AutomatedChecksExpandAllButton [](start = 29, length = 30)

Should we try to standardize both the names of the IDs ("AutomatedChecksExpandAllButton" has some context but "PauseButton" doesn't). Also, some (but not all) of these are sorted alphabetically. It might make it easier in the long run if we had a single approach (not necessarily part of this PR)

Reply:
Agreed. We are planning to do standardize/improve this in a future PR.